spec/controllers/api/v1/tags_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+describe Api::V1::TagsController do
+
+  describe "index" do
+
+    it "returns a json array of all tags" do
+      tag1 = create(:tag)
```

Comment:
two tags are created & both are used in expectations of the test, so seems like a useful assignment to me!